src/ape/cli/options.py

```diff
@@ -41,7 +41,7 @@ def project(self) -> ProjectManager:
             from ape import project
 
             self._project = project
-            self._project.config.load()
```

Comment:
I wonder if it would make sense to also have this class use the manager base class?
The class is `ApeCliContextObject` and is available fas a CLI automatic-pass-in option that basically contains managers, an `abort()` method and the `logger`.

Reply:
Can do! I am really loving this hack @evanblank3 is doing of injecting the managers into one place. Makes them really easy to access and reduce mutual dependencies!